build.gradle

```diff
@@ -68,6 +67,7 @@ allprojects {
             options.errorprone {
                 option('NullAway:AnnotatedPackages', 'com.palantir')
                 option('NullAway:CheckOptionalEmptiness', 'true')
+                option('AllSuggestionsAsWarnings') // https://github.com/google/error-prone/pull/3301
```

Comment:
This feature isn't yet released, but this should pick it up once we get the bump through gradle-baseline